index.html

```diff
@@ -770,7 +772,8 @@
       Storing and receiving JSON data is easy with serialization and deserialization.
     </p>
     <pre class="highlight">
-navigator.nfc.watch((message) => {
+const reader = new NFCReader();
+reader.watch(message => {
```

Comment:
Is this supposed to be `reader.onreading = message => ...` ?

Reply:
yes! Good catch